completion/available/lerna.completion.bash

```diff
@@ -1,19 +1,23 @@
-#!/bin/bash
+# shellcheck shell=bash
 # Lerna autocompletion.
-function _lerna_completions() {
-  local cur compls
 
-  # The currently-being-completed word.
-  cur="${COMP_WORDS[COMP_CWORD]}"
+if _command_exists lerna; then
+	function _lerna_completions() {
```

Comment:
Could we drop the plural and use double underscore prefix?

`__lerna_completion`

Reply:
fixed

Reply:
@NoahGorny almost. looks like it still only uses a single underscore prefix